diff --git a/Makefile b/Makefile
index ab26ae0..7dfba7c 100644
--- a/Makefile
+++ b/Makefile
@@ -1,10 +1,6 @@
 DMLC_CORE_PATH ?= ../dmlc-core
 DLPACK_INCLUDE_PATH ?= ../dlpack/include
 
-LDFLAGS = -pthread -lm
-CFLAGS =  -std=c++11 -Wall -O2\
-	 -Iinclude -I${DMLC_CORE_PATH}/include -I../include -I${DLPACK_INCLUDE_PATH}-Isrc -fPIC -fvisibility=hidden
-
 ifdef no_rtti
 	CFLAGS += -fno-rtti
 endif
 
